tests/src/test/java/alluxio/worker/AlluxioWorkerRestApiTest.java

```diff
@@ -91,6 +97,37 @@ public void getCapacityBytesTest() throws Exception {
         NO_PARAMS, HttpMethod.GET, capacityBytes).run();
   }
 
+  private Configuration mockConfiguration() {
```

Comment:
I'm not familiar with the mock and whitebox mechanism, the current solution is brought from AlluxioMasterRestApiTest, what's the downside of powermock? Is it related to performance?